avm/src/vm/memory.rs

```diff
@@ -640,15 +638,32 @@ impl HandlerMemory {
           }
         }
       }
-      dest.mems.append(&mut orig_arr_copies);
+      if cfg!(debug_assertions) {
```

Comment:
What is this?

Reply:
evaluates to `true` on debug builds and `false` on release builds